testing/catalog.go

```diff
@@ -815,6 +815,19 @@ func (c *Catalog) DefaultBOSHDeploymentWithOps(name, manifestRef string, opsRef
 	}
 }
 
+// DefaultBOSHDeploymentWithUnsupportedOps fissile deployment CR with ops
```

Comment:
I don't know why there is fissile deployment? do we have any history about it?

Reply:
I think it was a plain copy paste of a existing funcs, will take a look